### DISCUSSION TITLE
[Q&A] 如何不通过自定义模型方式手动修改默认的 gpt-4-1106-preview，gpt-4-32k 模型到 GPT 的其他版本号上

### DISCUSSION THREAD

**Discussion by itq5** [lobehub/lobehub#420] [Self Hosting] [answered]
### 🧐 问题描述 | Proposed Solution

[Question] 如何手动修改默认的gpt-4-1106-preview，gpt-4-32k模型到GPT的其他版本号上，不通过自定义模型方式，需要改代码哪个位置，谢谢！
![image](https://github.com/lobehub/lobe-chat/assets/42521985/1a135d23-13d7-47c0-a7ab-68c1e445078f)

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
现在暂不支持指定，你要改的话，要自己 fork 了修改：

https://github.com/lobehub/lobe-chat/blob/main/src/types/llm.ts#L4

```diff
-  GPT4 = 'gpt-4',
+  GPT4 = 'gpt-4-1106-preview',
```
